packages/board-server/tests/integration/board-api.test.ts

```diff
@@ -0,0 +1,494 @@
+import { suite, test } from 'node:test';
+import assert from 'node:assert';
+import { request } from 'node:http';
+import { startServer, stopServer } from "../../src/server.js";
+
+import fs from 'fs';
+import path, { resolve } from 'path';
+
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+
+import { createAccount } from '../../src/server/store.js'
+
+let serverInstance: { server: any; port: string | number };
+
+var account: { api_key: any; account?: string; };
+
+import { deepStrictEqual } from 'assert';
+
+const MODULE_PATH = dirname(fileURLToPath(import.meta.url));
+const ROOT_PATH = resolve(MODULE_PATH, "../../../");
+
+test.before(async () => {
+  serverInstance = await startServer(ROOT_PATH);
+  account = await createAccount('test')
+});
+
+test.after(async () => {
+  await stopServer(serverInstance.server);
+  console.log('Server stopped');
+  process.exit(0);
+});
+
+process.on('uncaughtException', async (err) => {
+  console.error('Uncaught Exception:', err);
+  if (serverInstance) {
+    await stopServer(serverInstance.server);
+  }
+  process.exit(1);
+});
+
+process.on('unhandledRejection', async (reason) => {
+  console.error('Unhandled Rejection:', reason);
+  if (serverInstance) {
+    await stopServer(serverInstance.server);
+  }
+  process.exit(1);
+});
+
+function readBoard(name: string) {
+      // Get the directory name
```

Comment:
indentation (running the formatter should probably pick this up)

Reply:
Thank you!